authors/apps/articles/tests/test_search.py

```diff
@@ -0,0 +1,38 @@
+from django.urls import reverse
+
+from rest_framework import status
+
+from authors.apps.articles.tests.base_class import ArticlesBaseTest
+from .test_data import valid_article
+from ..models import Article
+
+
+class TestSearchArticle(ArticlesBaseTest):
+
+    def test_search_by_keyword(self):
+        """
+        Tests if a user can search articles by keyword
+        """
+        self.add_article()
+        response = self.client.get(self.articles_url+"?keyword=the")
+        print(response.data['articles'][0])
```

Comment:
Hey @BagzieGracious , thanks for the good work, However on line 18, I think you needed the print for debugging purposes. You could consider removing it if that's the case.